front-end/src/UserHomePage.js

```diff
@@ -0,0 +1,121 @@
+import * as React from "react";
+import TopNavigation from "@cloudscape-design/components/top-navigation";
+import Container from "@cloudscape-design/components/container";
+import Header from "@cloudscape-design/components/header";
+import ContentLayout from "@cloudscape-design/components/content-layout";
+import SpaceBetween from "@cloudscape-design/components/space-between";
+import Button from "@cloudscape-design/components/button";
+import Badge from "@cloudscape-design/components/badge";
+
+export default function UserHomePage (){
+  const [data, setData] = React.useState([
+    { title:"Employee survey 1", due: ('2023-10-20'),id:1},
```

Comment:
The spacing here is wrong. Should be something like

```
{ title: "Employee survey 1", due: ('2023-10-20'), id:1 }
```

Created a new task for you. https://internalsurveytool.atlassian.net/browse/SCRUM-14